packages/components/.storybook/main.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  stories: ["../src", "../docs"],
+  stories: ["../docs", "../src"],
```

Comment:
Just swapped docs and src here, so that the "docs" stories (typograhpy and tokens) show up first.

No reason to do that, but it felt weird having them buried at the bottom.

Reply:
I like having the tokens first, having the components nested under a section named `src` feels a bit awkward. That's just with `stories: ["../"],` right?

Reply:
> That's just with stories: ["../"], right?

That's right

Reply:
Could alternatively make this change (putting docs/ first) without moving to auto-generated titles.

Reply:
I think if the choice is between auto-generated titles and nested stories, I'm more down for auto-generated titles